lib/screens/create_region_screens/add_region_description/add_region_description.dart

```diff
@@ -10,9 +10,33 @@ import 'package:seeds/domain-shared/ui_constants.dart';
 import 'package:seeds/screens/create_region_screens/viewmodels/create_region_bloc.dart';
 import 'package:seeds/utils/build_context_extension.dart';
 
-class AddRegionDescription extends StatelessWidget {
+class AddRegionDescription extends StatefulWidget {
   const AddRegionDescription({Key? key}) : super(key: key);
 
+  @override
+  _AddRegionDescription createState() => _AddRegionDescription();
+}
+
+class _AddRegionDescription extends State<AddRegionDescription> {
+  final _keyController = TextEditingController();
+  late CreateRegionBloc _createRegionBloc;
+
+  @override
+  void initState() {
+    super.initState();
+    _createRegionBloc = BlocProvider.of<CreateRegionBloc>(context);
+    _keyController.text = _createRegionBloc.state.regionDescription ?? '';
+    _keyController.addListener(() {
+      setState(() {});
+    });
```

Comment:
```
    _keyController.addListener(() {
        BlocProvider.of<CreateRegionBloc>(context).add(OnRegionDescriptionChange(_keyController.text));
    });
```

Reply:
Also you can user a `CreateRegionBloc _createRegionBloc` variable then initialize it in the initState to avoid write all the `BlocProvider.of<CreateRegionBloc>(context)`